install/kubernetes/cilium/charts/operator/templates/deployment.yaml

```diff
@@ -38,6 +38,7 @@ spec:
 {{- if .Values.global.kubeConfigPath }}
         - --k8s-kubeconfig-path={{ .Values.global.kubeConfigPath }}
 {{- end }}
+        - --synchronize-k8s-nodes={{ .Values.synchronizeK8sNodes }}
         command:
         - cilium-operator
         env:
```

Comment:
needs a regeneration of `quick-install.yaml` in the `install/kubernetes` directory.